CHANGELOG.md

```diff
@@ -4,9 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Master branch
+## 0.14.8
```

Comment:
Should this be `0.15.0`? Due to dropping py36 support...

Reply:
💯